faq.rst

```diff
@@ -0,0 +1,46 @@
+Frequently Asked Questions
+==========================
+
+The following FAQs are organized into
+
+- a "Components" section that includes each of the seven PURL components
+  (``scheme``, ``type``, ``namespace``, ``name``, ``version``, ``qualifiers``
+  and ``subpath``), and
+
+- a "General" section containing a mix of questions and answers that don't fit
+  neatly into a component-focused category.
+
+If you have a question about the PURL specification and don't find an answer
+below, you can open an issue `here <https://github.com/package-url/purl-spec/issues/new?template=Blank+issue>`_.
```

Comment:
I would probably ditch this paragraph since it's stating something that is hopefully obvious to anyone interacting with this repository.

Reply:
@mprpic   I wholeheartedly agree.  🙂  Will do as soon as we clarify whether to retain (temporarily or otherwise) the current lowercase `purl` usage in the core spec.  (See my [comment further below](https://github.com/package-url/purl-spec/pull/361#issuecomment-2640836146) to you and @pombredanne re Ecma TC54-TG2's approval of 'PURL'/'Package-URL'.)

Reply:
Removed para re opening an issue.